lib/vmpooler/pool_manager.rb

```diff
@@ -459,69 +458,17 @@ def _check_snapshot_queue
       end
     end
 
-    def migration_limit(migration_limit)
```

Comment:
As the migration limit is now a per vSphere provider setting the documentation should be updated too.

Reply:
It probably should be a per vSphere provider setting, but it's still tracking migrations in redis and getting the setting from the global configuration section instead of checking the provider for this setting. I could add this change here. I was considering tackling that and the task_limit concept in a future change.

Reply:
Could be as simple as add in the docs `migration limit (Only affects vSphere Provider)`

Reply:
Oh I see what you're saying, not that it's a per provider setting, but rather that it is only a vsphere provider setting.

Reply:
yeah...  The docs suggest its global but no longer.

Reply:
I added a mention in the example configuration file. I couldn't find another reference to migration_limit, please let me know if I've missed another location this should go.